content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -332,16 +330,13 @@ $ tree .dvc/cache
     └── 0b40427ee0998e9802335d98f08cd98f
 ```
 
-The cache file with `.dir` extension is a special text file that contains the
-mapping of files in the `data/` directory (as a JSON array), along with their
-hash values. The other two cache files are the files inside `data/`.
+This `.dir` file contains the mapping of files in `data/images` (as a JSON
+array), including their hash values. That's how DVC knows that the other two
+cached files belong in the directory:
 
-A typical `.dir` cache file looks like this:
-
-```dvc
```

Comment:
No reason to change this

Reply:
The sentence was unnecessary as it's already mentioned in the previous paragraph. The code block I have rolled back now.